examples/text_to_image/train_text_to_image_lora.py

```diff
@@ -0,0 +1,824 @@
+# coding=utf-8
+# Copyright 2023 The HuggingFace Inc. team. All rights reserved.
```

Comment:
(nit)

maybe also mention the original lora repo in copyright.

Reply:
[This](https://github.com/microsoft/LoRA) or [this](https://github.com/cloneofsimo/lora)?

Reply:
Don't think there is really an original repo we copied from - we just took the idea. Wouldn't change anything here.